ImmuDB4Net/proto/schema.proto

```diff
@@ -30,82 +30,131 @@ message Key {
 }
 
 message Permission {
+  // Database name
```

Comment:
Something's weird with the indentation here - is it also in the original .proto file ?

Reply:
done